src/js/api.js

```diff
@@ -8,11 +8,28 @@ import mime from "rest/interceptor/mime";
 import errorCode from "rest/interceptor/errorCode";
 import defaultRequest from "rest/interceptor/defaultRequest";
 import hateoas from "rest/interceptor/hateoas";
-import basicAuth from "rest/interceptor/basicAuth";
+import interceptor from "rest/interceptor";
+
+export const AUTH_TOKEN_PROPNAME = "readinglist:auth:token";
```

Comment:
Is "Propname" a common naming for storage entries ? I would have said *keyname* for instance :)